### PR TITLE
feat(Navigation): add iconSize for navigation icons, update margin between icon and text

### PR DESCRIPTION
**Pull request by aeksandla**
…tween icon and text

### CODE REVIEW

File: src/navigation/components/Header/Header.tsx [hunk 1]
```diff
@@ -52,11 +57,32 @@ const MobileMenuButton: React.FC<MobileMenuButtonProps> = ({
...
 export const Header: React.FC<HeaderProps> = ({data, logo}) => {
-    const {leftItems, rightItems} = data;
+    const {leftItems, rightItems, iconSize} = data;
     const [isSidebarOpened, setIsSidebarOpened] = useState(false);
     const [activeItemId, setactiveItemId] = useState<string | undefined>(undefined);
 
+    const getNavigationItemWithIconSize = useCallback(
+        (item: NavigationItemModel) => {
+            if (!(iconSizeKey in item)) {
+                return {...item, iconSize};
```
Comment (yuberdysheva) [inline src/navigation/components/Header/Header.tsx:70]: What happens if iconSize is not set? Maybe you should add the default value

**Review by yuberdysheva — CHANGES_REQUESTED** [review]

File: src/navigation/components/Header/Header.tsx (hunk 1 above)
Comment (aeksandla) [reply src/navigation/components/Header/Header.tsx:70]: If iconSize is not set, then no styles will be applied to the image.
seeIconSizeStyle: https://github.com/gravity-ui/page-constructor/pull/329/files#diff-01cc3be42ecfb8f27579002a8ce8d95d1aa53569e00008dd1510400ed43425e7R18

Comment (yuberdysheva) [reply src/navigation/components/Header/Header.tsx:70]: Maybe we should set some default size?

File: src/navigation/components/NavigationItem/components/ContentWrapper/ContentWrapper.scss [hunk 2]
```diff
@@ -7,7 +7,7 @@ $block: '.#{$ns}content-wrapper';
         display: flex;
         width: 20px;
         height: 20px;
```
Comment (yuberdysheva) [inline src/navigation/components/NavigationItem/components/ContentWrapper/ContentWrapper.scss:9]: can we delete this styles with default iconSize?

**Review by yuberdysheva — APPROVED** [review]

File: src/navigation/components/NavigationItem/components/ContentWrapper/ContentWrapper.scss (hunk 2 above)
Comment (aeksandla) [reply src/navigation/components/NavigationItem/components/ContentWrapper/ContentWrapper.scss:9]: of course

**Review by yuberdysheva — APPROVED** [review]

**Pull request merged**